test.js

```diff
@@ -0,0 +1,45 @@
+var tape = require('tape')
+var signalhub = require('signalhub')
+var swarm = require('./')
+var wrtc = require('electron-webrtc')()
+
+tape('greet and close', function (t) {
+  t.plan(5)
+
+  var peer1 = swarm(signalhub('swarmtest', 'https://signalhub.mafintosh.com'), {wrtc})
```

Comment:
could you make `swarmtest` a random string? so it doesn't conflict when two people are running the tests